radio/src/thirdparty/FatFs/ffconf.h

```diff
@@ -79,7 +79,7 @@
 / Locale and Namespace Configurations
 /---------------------------------------------------------------------------*/
 
-#define FF_CODE_PAGE	437
+#define FF_CODE_PAGE 437
```

Comment:
Unnecessary white space change here

Reply:
/---------------------------------------------------------------------------*/


#define FF_CODE_PAGE    437
/* This option specifies the OEM code page to be used on the target system.